src/cpp/src/utils.cpp

```diff
@@ -704,6 +704,35 @@ ov::Tensor merge_text_and_image_embeddings_llava(const ov::Tensor& input_ids, ov
     return inputs_embeds;
 }
 
+ov::Tensor merge_text_and_image_embeddings_nanollava(const ov::Tensor& input_ids, ov::Tensor& text_embeds, const std::vector<ov::Tensor>& image_embeds, int64_t image_tok) {
```

Comment:
`merge_text_and_image_embeddings_llava()` is in utils because it's shared between models. `merge_text_and_image_embeddings_nanollava()` is unique to nanolava. Move it to classes.cpp

Reply:
Done.